<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>743</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1391</width>
    <height>743</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>1051</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background :rgb(255, 229, 237) ;border-radius : 10;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1401</width>
     <height>761</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>961</width>
     <height>761</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background :qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 246, 254, 255), stop:1 rgba(255, 255, 255, 255));border-radius : 20;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ex_bt">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>700</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(102, 112, 98);
border-radius : 10px;
color:white}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_bt">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>30</y>
     <width>91</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(102, 112, 98);
border-radius : 10px;
color:white}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tw_ed">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color:rgb(255, 255, 255);border-radius : 6px;
color:black}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>WEB Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background :rgb(255, 219, 229);border-radius : 20;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>1351</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : rgb(190, 210, 255) ;border-radius : 10;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="start_tm">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::TimeZone</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string> Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>2022-03-23 19:09:32</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="stop_tm">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::TimeZone</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="list_tweet">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>251</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar
{
border: solid grey;
border-radius: 15px;
color: black;
}
QProgressBar::chunk 
{
background-color: #05B8CC;
border-radius :15px;
}      </string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>bg_3</zorder>
  <zorder>bg</zorder>
  <zorder>bg_2</zorder>
  <zorder>tableView</zorder>
  <zorder>ex_bt</zorder>
  <zorder>search_bt</zorder>
  <zorder>tw_ed</zorder>
  <zorder>label</zorder>
  <zorder>start_tm</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>time</zorder>
  <zorder>label_4</zorder>
  <zorder>stop_tm</zorder>
  <zorder>list_tweet</zorder>
  <zorder>progressBar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
